File: b/Python/Processor.py
class Processor:
	# Konstruktor
	def __init__(self):
		self.__name = ""
		self.__price = 0

	# Setter & Getter
	def setName(self, name):
		self.__name = name

	def setPrice(self, price):
		self.__price = price

	def getName(self):
		return self.__name

	def getPrice(self):
		return self.__price

	# Method
	def printProcessor(self):
		print("- PROCESSOR -")
		print("Name        :", self.__name)

File: b/Java/Processor.java
public class Processor {
	// Atribut
	private String name;
	private int price;

	// Konstruktor
	public Processor() {
		name = "";
		price = 0;
	}

	// Setter & Getter
	public void setName(String name) {
		this.name = name;
	}
	public void setPrice(int price) {
		this.price = price;
	}

	public String getName() {
		return name;
	}
	public int getPrice() {
		return price;
	}

	// Method
	public void printProcessor() {
		System.out.println("- PROCESSOR -");
		System.out.println("Name        : " + name);
	}
}